src/zh/mh1234/src/eu/kanade/tachiyomi/extension/zh/mh1234/MH1234.kt

```diff
@@ -0,0 +1,139 @@
+        }
+    }
+
+    // Popular Page
+
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/comic/one/page_hit.html")
+
+    override fun popularMangaSelector() = ".itemBox"
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        thumbnail_url = element.selectFirst("img")?.attr("src")
+        element.selectFirst(".title")!!.let {
+            title = it.text()
+            this.setUrlWithoutDomain(it.absUrl("href"))
+        }
+    }
+
+    override fun popularMangaNextPageSelector() = null
+
+    // Latest Page
+
+    override fun latestUpdatesRequest(page: Int) = GET("$baseUrl/comic/one/page_recent.html")
+
+    override fun latestUpdatesSelector() = popularMangaSelector()
+
+    override fun latestUpdatesFromElement(element: Element) = popularMangaFromElement(element)
+
+    override fun latestUpdatesNextPageSelector() = null
+
+    // Search Page
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) = GET(
+        baseUrl.toHttpUrl().newBuilder()
+            .addPathSegment("search/")
+            .addQueryParameter("keywords", query)
+            .addQueryParameter("page", page.toString()).build(),
+    )
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val doc = response.asJsoup()
+        val cur = response.request.url.queryParameter("page")?.toInt() ?: 1
+        val total = doc.selectFirst("#total-page")!!.attr("value").toInt()
+        val mangas = doc.select(popularMangaSelector()).map(::popularMangaFromElement)
+        return MangasPage(mangas, cur < total)
+    }
+
+    override fun searchMangaSelector() = throw UnsupportedOperationException()
+
+    override fun searchMangaFromElement(element: Element) = throw UnsupportedOperationException()
+
+    override fun searchMangaNextPageSelector() = throw UnsupportedOperationException()
+
+    // Manga Detail Page
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        val txtItme = document.select(".txtItme")
+        author = txtItme[0].text()
+        status = when (txtItme[2].selectFirst("a:last-child")?.text()) {
+            "连载中" -> SManga.ONGOING
+            "已完结" -> SManga.COMPLETED
+            else -> SManga.UNKNOWN
+        }
+        val desc = document.selectFirst("#full-des") ?: document.selectFirst("#simple-des")
+        description = desc?.text()?.substring(3)
+    }
+
+    // Manga Detail Page / Chapters Page (Separate)
+
+    // override fun chapterListRequest(manga: SManga) = GET(baseUrl + manga.url)
+
+    override fun chapterListSelector() = throw UnsupportedOperationException()
+
+    override fun chapterFromElement(element: Element) = throw UnsupportedOperationException()
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val doc = response.asJsoup()
+        val date = DATE_FORMAT.parse(doc.selectFirst(".txtItme .date")!!.text())?.time
```

Comment:
```suggestion
        val date = DATE_FORMAT.tryParse(doc.selectFirst(".txtItme .date")?.text())
```
need to import, and inline it